File: client/jetbrains/src/main/java/com/sourcegraph/cody/context/NoGitRepositoryEmbeddingStatus.java
package com.sourcegraph.cody.context;

import com.intellij.openapi.project.Project;
import com.sourcegraph.cody.Icons;
import javax.swing.Icon;
import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

public class NoGitRepositoryEmbeddingStatus implements EmbeddingStatus {
  @Override
  public @Nullable Icon getIcon() {
    return Icons.Repository.Missing;
  }

  @Override
  public @NotNull String getTooltip(@NotNull Project project) {
    return "No Git repository opened";
  }

  @Override
  public @NotNull String getMainText() {
    return "";
  }
}

File: client/jetbrains/src/main/java/com/sourcegraph/cody/context/NoGitRepositoryEmbeddingStatus.kt
package com.sourcegraph.cody.context

import com.intellij.openapi.project.Project
import com.sourcegraph.cody.Icons
import javax.swing.Icon

class NoGitRepositoryEmbeddingStatus : EmbeddingStatus {
  override fun getIcon(): Icon? {
    return Icons.Repository.Missing
  }

  override fun getTooltip(project: Project): String {
    return "No Git repository opened"
  }

  override fun getMainText(): String {
    return ""
  }
}
